### DISCUSSION TITLE
What did Prometheus steal from Olympus?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1157] [Galaxy Trivia] [answered]
**Bonus:** What is the number 55 in Bingo?

*answer with emojis*

**Comment by tomerf** [comment, marked as answer]
Prometheus stole fire from the Olympian gods.

Bonus: 🐍🐍
The answer is "snakes alive", not sure how to describe the second word, maybe: 🐍🐍🚫💀
